deployments/pod-sriovdp.yaml

```diff
@@ -9,6 +9,8 @@ spec:
     imagePullPolicy: IfNotPresent
     command: [ "/bin/sh", "-c", "--" ]
     args: [ "sriovdp --logtostderr 10;" ]
+    securityContext:
+      privileged: true
```

Comment:
why this is required? what  is required privileged

Reply:
The device plugin daemonset needs to be run  in privilege mode because it needs to query host forwarding table. We also need to enable this should we add features to watch Link status. 

Have you managed to run this with `privileged: true`?